internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -2015,16 +2016,36 @@ func Install(runtime.Sequence, any) (runtime.TaskExecutionFunc, string) {
 
 			var disk string
 
-			disk, err = r.Config().Machine().Install().Disk()
+			matchExpr, err := r.Config().Machine().Install().DiskMatchExpression()
 			if err != nil {
-				return err
+				return fmt.Errorf("failed to get disk match expression: %w", err)
+			}
+
+			switch {
+			case matchExpr != nil:
+				logger.Printf("using disk match expression: %s", matchExpr)
```

Comment:
Log intended or left from development only? Should be useful I believe

Reply:
yes, exactly, to provide feedback to the user